cli/template/extras/src/server/api/routers/example/with-auth-drizzle.ts

```diff
@@ -0,0 +1,24 @@
+import { z } from "zod";
+import {
+  createTRPCRouter,
+  publicProcedure,
+  protectedProcedure,
+} from "~/server/api/trpc";
+
+export const exampleRouter = createTRPCRouter({
+  hello: publicProcedure
+    .input(z.object({ text: z.string() }))
+    .query(({ input }) => {
+      return {
+        greeting: `Hello ${input.text}`,
+      };
+    }),
+
+  getAll: publicProcedure.query(({ ctx }) => {
+    return ctx.db.query.example.findMany();
```

Comment:
is this correct?

Reply:
yep 😎 

<img width="1132" alt="CleanShot 2023-06-04 at 21 33 54@2x" src="https://github.com/t3-oss/create-t3-app/assets/51714798/e3ffe2a3-64d4-4d77-81d4-b9eafdd12c86">